src/main/java/org/dave/compactmachines3/gui/framework/widgets/WidgetButton.java

```diff
@@ -63,7 +63,7 @@ public void setUnlocalizedLabel(String unlocalizedLabel) {
 
     @Override
     public void draw(GuiScreen screen) {
-        //Logz.info("Width: %d, height: %d", width, height);
+        //Logz.info("Width: {}, height: {}", width, height);
```

Comment:
Prefer full removal of unnecessary logging

Reply:
Noted